Dockerfile

```diff
@@ -14,7 +14,9 @@ FROM alpine:latest
 
 RUN apk update && \
     apk upgrade && \
-    apk add ca-certificates git openssh-client python3 py3-pip tini && \
+    apk add --no-cache ca-certificates git openssh-client python3 py3-pip tini && \
+    python3 -m venv /venv && \
+    . /venv/bin/activate && \
```

Comment:
#700 already has a fix for the above problem in Dockerfile. I think we should treat this as a separate issue to be addressed by #700. If you agree, can you please undo this change in Dockerfile, and keep this PR for fixing lint? @0luwaj0x1

Reply:
Agreed